src/main/java/org/jenkinsci/plugins/workflow/graphanalysis/NodeStepNamePredicate.java

```diff
@@ -63,11 +65,13 @@ public boolean apply(@Nullable FlowNode input) {
                 // Workaround for cases where someone is using the latest workflow-api which has StepNode
                 //  but manages to skip the post-2.26 versions of workflow-cps where it a parent of workflow-cps StepNode.
                 // Technically consumers *were* supposed to call ForkScanner#setParallelStartPredicate, but better to be foolproof.
-                Method getDescriptorMethod = input.getClass().getDeclaredMethod("getDescriptor", null);
+                Method getDescriptorMethod = input.getClass().getMethod("getDescriptor", null);
```

Comment:
@jglick Yes, I suppose I could -- this just makes it more explicit, but isn't really needed.